### ISSUE TITLE
warping stuck

### ISSUE THREAD

**Issue by john5514** [suriyun-production/mmorpg-kit-docs#2295]
Hello,

if I die anywhere warping ui comes up and gets stuck in the warping text I tried deleting it I can at least move but I can't do any attack or anything other than walking.

**Comment by insthync** [comment]
How ?

**Comment by insthync** [comment]
Video please

**Comment by john5514** [comment]
https://www.veed.io/view/d74f7208-fb92-4fc8-86c1-f6f91a1d29bb?panel=share

here

**Comment by john5514** [comment]
> Video please

i try database update but not working

**Comment by insthync** [comment]
I guess the problem occurring because it cannot connect to another map server, did you check the map-server log?

**Comment by john5514** [comment]
i looked, but something didn't catch my eye

https://we.tl/t-MPEE2Ewp55

**Comment by john5514** [comment]
it does not enter some maps, it gives server rejected error and kicks out of the game and I haven't changed anything, everything was like this while everything was working smoothly

separately, if the mysql error is new in some of the logs, I delete the config and switch it on and off to restart it

If you see them, ignore them if they are not new

**Comment by insthync** [comment]
Which map your current will be warped to ?

**Comment by insthync** [comment]
It have so many `Unable to connect to any of the specified MySQL hosts` error, is it able to connected to MySQL ?

**Comment by insthync** [comment]
It is still spawning Map001 server, but you did not set it to scenes in build, and it shows "Scene 'Map001' couldn't be loaded because it has not been added to the build settings or the AssetBundle has not been loaded."

**Comment by insthync** [comment]
Map002 too

**Comment by insthync** [comment]
Seems like you set wrong parameter, it should not start map-server if it is going to be launched later by map-spawner-server

**Comment by insthync** [comment]
In case you doubt how can I know that you put wrong parameters

![Screenshot_911](https://github.com/suriyun-production/mmorpg-kit-docs/assets/2697550/8b068880-29ba-472a-8334-0c6b17f3caa3)

**Comment by insthync** [comment]
So many mistakes, but start from start-servers parameters, which part of the document made you understand the run-server parameter incorrectly?

**Comment by john5514** [comment]
I set the parameter in this way before, there was no problem, this is the parameter


mainmap lobby map and respawn maps respawn on some maps, some maps do not respawn at all 

and some maps do not go at all server rejected says 

we use sqlite3 not mysql host 


parameter "D:\MMO\Might of Empire.exe" -startMapSpawnServer -spawnExePath "D:\MMO\Might of Empire.exe" -startCentralServer -startDatabaseServer -startDatabaseManager -startMapServer -startDatabaseManager -machineAddress "185.249.xxxxx"

**Comment by john5514** [comment]
> Map001

It's impossible not to have Map001 installed, 

because I think it wouldn't teleport unless the main server map was loaded.

**Comment by john5514** [comment]
> Yanlış parametre ayarlamışsınız gibi görünüyor, daha sonra harita-yumurtlama sunucusu tarafından başlatılacaksa harita-sunucusunu başlatmamalıdır.

Is it also important to write in paramater?

**Comment by john5514** [comment]
i think i found https://prnt.sc/5oPA9J56W-bT this correct mapspawn 

this is the broken map spawn option, I said I would look here, I looked 4-5 times, I felt that something was missing, I opened the old project, these places do not work

https://prnt.sc/nyURMOA4dhfY

**Comment by john5514** [comment]
I don't understand why this place went while I didn't touch anything, but I will copy the MapSpawnNetworkManager from the old project to the new project, if it works, I will forward it.

**Comment by john5514** [comment]
even though I copied it, I still can't find the place to insert the map, but I pulled it from the working place

**Comment by insthync** [comment]
Setup config in config/serverConfig.json file.

**Comment by insthync** [comment]
> > Map001
> 
> It's impossible not to have Map001 installed,
> 
> because I think it wouldn't teleport unless the main server map was loaded.

![Screenshot_912](https://github.com/suriyun-production/mmorpg-kit-docs/assets/2697550/03e8aab7-2278-4a85-abbe-88380b04b921)

???

**Comment by insthync** [comment]
If you are going to use SQLite then you might setup it incorrectly because it is obviously show that it is trying to connect to MySQL

**Comment by john5514** [comment]
> > > Map001
> > 
> > 
> > It's impossible not to have Map001 installed,
> > because I think it wouldn't teleport unless the main server map was loaded.
> 
> ![Screenshot_912](https://user-images.githubusercontent.com/2697550/278809840-03e8aab7-2278-4a85-abbe-88380b04b921.png)
> 
> Correct?, If you don't going to spawn these maps, then remove it from the list

The problem is that this part does not come to me, I copied and pasted the whole mmorpg kit folder from the working one and did not come to the non-working one again, shouldn't this part be?

**Comment by insthync** [comment]
> I set the parameter in this way before, there was no problem, this is the parameter
> 
> mainmap lobby map and respawn maps respawn on some maps, some maps do not respawn at all
> 
> and some maps do not go at all server rejected says
> 
> we use sqlite3 not mysql host
> 
> parameter "D:\MMO\Might of Empire.exe" -startMapSpawnServer -spawnExePath "D:\MMO\Might of Empire.exe" -startCentralServer -startDatabaseServer -startDatabaseManager -startMapServer -startDatabaseManager -machineAddress "185.249.xxxxx"

This is incorrect because it has `-startMapServer`

**Comment by john5514** [comment]
> If you are going to use SQLite then you might setup it incorrectly because it is obviously show that it is trying to connect to MySQL

If you start with the config files not created, if you saw this message at first, it does not create the config file for sqlite3 without turning it off and on because you created it from scratch at first, this error is corrected when you turn it off and on.

I told you not to care if you see it early in the logs, 

I will look if the message is new

**Comment by insthync** [comment]
> > > > Map001
> > > 
> > > 
> > > It's impossible not to have Map001 installed,
> > > because I think it wouldn't teleport unless the main server map was loaded.
> > 
> > 
> > ![Screenshot_912](https://user-images.githubusercontent.com/2697550/278809840-03e8aab7-2278-4a85-abbe-88380b04b921.png)
> > Correct?, If you don't going to spawn these maps, then remove it from the list
> 
> The problem is that this part does not come to me, I copied and pasted the whole mmorpg kit folder from the working one and did not come to the non-working one again, shouldn't this part be?

I don't get what you want to tell me, but it is incorrect, that is it.

But you don't have to look at it because it will use all config from `config/serverConfig.json` only, it won't use anything from component anymore if `serverConfig.json` was created.

**Comment by john5514** [comment]
> > > > > Harita001
> > > > 
> > > > 
> > > > Map001'in kurulu olmaması mümkün değil 
> > > > çünkü ana sunucu haritası yüklenmediği sürece ışınlanmayacağını düşünüyorum.
> > > 
> > > 
> > > ![Ekran görüntüsü_912](https://user-images.githubusercontent.com/2697550/278809840-03e8aab7-2278-4a85-abbe-88380b04b921.png)
> > > Doğru mu?, Bu haritaları oluşturmayacaksanız listeden kaldırın
> > 
> > 
> > Sorun şu ki bu kısım bana gelmiyor, çalışan olandan mmorpg kit klasörünün tamamını kopyalayıp yapıştırdım ve çalışmayan tarafa yine gelmedi, bu kısım olması gerekmez mi?
> 
> Bana ne söylemek istediğini anlamadım ama bu yanlış, bu kadar.
> 
> Ancak ona bakmanıza gerek yok çünkü tüm yapılandırmayı yalnızca kullanacak , eğer oluşturulduysa `config/serverConfig.json`artık bileşenden hiçbir şey kullanmayacak .`serverConfig.json`

Where you add maps in the editor, I don't have this list that is in the spawing maps. 

I can't add to or remove from the list, I can't add to or remove from the config, yes I know I can fix it I can fix it from within "config", but doesn't the fact that it goes here indicate that something is wrong?

https://prnt.sc/q6RZSMgTNXAp

I don't have the part inside the red circle

this is my old project, this part does not come blank in the new project

new projecet

https://prnt.sc/tn7XfHAqcxHi where? map list for add remove build

**Comment by john5514** [comment]
How will the first build pull them in without this place? into the config file

**Comment by insthync** [comment]
I still don't get what you want to tell me

**Comment by john5514** [comment]
Can you look at the pictures, I don't have the complete list in these pictures, I think this problem occurred after this went,

My paramatres were always the same, always working, always reborn, no problems.

**Comment by insthync** [comment]
> Can you look at the pictures, I don't have the complete list in these pictures, I think this problem occurred after this went,
> 
> My paramatres were always the same, always working, always reborn, no problems.

No, It must not start a map-server if all servers will be started by map-spawn server

**Comment by insthync** [comment]
I looked at the picture, I saw it have Map001, Map002 in the list. It should not be there isn't it?

**Comment by john5514** [comment]
> Resme baktım listede Map001, Map002 olduğunu gördüm. Orada olmaması gerekir değil mi?

yes, they are not in that list because this is my old project, they are not even added in the new config, since they remain in the editor, it is constantly looking for them, because they are added in the build list, but I cannot update the build list except config.

**Comment by insthync** [comment]
Show me your `serverConfig.json`

**Comment by john5514** [comment]
https://prnt.sc/OMUW-auW5FO8

**Comment by insthync** [comment]
Look your character's respawn map in database, try find the tool to do it via google (may find "SQLiteBrowser")

**Comment by john5514** [comment]
which table?

**Comment by john5514** [comment]
https://prnt.sc/qESH8UJ2ielf
true? table

**Comment by insthync** [comment]
Yes, that table

**Comment by john5514** [comment]
https://prnt.sc/Ckr2cIQN0Rl7

**Comment by john5514** [comment]
If you say that it is relevant here, for example, when you die in the lavahill map, it throws it back to the mainmap

**Comment by insthync** [comment]
Okay, then the problem occurred in the map in the video only?

**Comment by insthync** [comment]
What is the map in the video?

**Comment by john5514** [comment]
> Videodaki harita nedir?

    "FarmMap001",

**Comment by insthync** [comment]
I want to see `FarmMap001`'s map info

**Comment by john5514** [comment]
https://www.veed.io/view/38fe274e-eb33-4ac0-b9df-6093ea9efd9d?sharingWidget=true&panel=share

new video

**Comment by john5514** [comment]
> `FarmMap001`adlı kişinin harita bilgilerini görmek istiyorum

https://prnt.sc/uk9yj8B5WUF8
https://prnt.sc/AseNigUTpz86

**Comment by insthync** [comment]
![Screenshot_913](https://github.com/suriyun-production/mmorpg-kit-docs/assets/2697550/e67d4636-00d3-4d24-b4e5-9e9ee10b0f8e)

Do you intend to move player to instance?

**Comment by john5514** [comment]
they've always been like this.

**Comment by john5514** [comment]
now yes no problem, just started teleporting to maps that didn't teleport before making these settings " while giving server rejected error"


even though I didn't change anything, I didn't understand 


respawn is working, 

separately, when I try on mobile, the character does not run, but when I press shifte in the editor, it runs

**Comment by insthync** [comment]
In this post we will talk about teleport issues only, if you have other issues, please create a new post.

**Comment by insthync** [comment]
Do you know differences of default and enter instance?

**Comment by john5514** [comment]
> Varsayılan ve örnek girme arasındaki farkları biliyor musunuz?

entrance is just the portal, you need to log in, I guess in default, you don't need to enter directly, right?

if you die, it respawns in default, it doesn't respawn in entrance, I think you need to go to the portal

**Comment by insthync** [comment]
When enter instance it will enter to a new instance of map server which will have to wait a new map-server to be spawned if it has no available pooled yet.

**Comment by john5514** [comment]
> Kapalı, yanıt vermediği için sorun açık bir şekilde örneklenmiş haritaya ışınlanacak ancak örneklenmiş harita henüz ortaya çıkmadığından örnekli harita sunucusunun oluşmasını beklemek zorunda, bu yüzden ışınlanma ayarını değiştirin varsayılana ayarlamak sorunu çözebilir.

thank you for helping no here any problem for now sir good day
